jsonnet/lib/karabiner.libsonnet

```diff
@@ -90,4 +90,16 @@
     type: 'frontmost_application_' + type,
     bundle_identifiers: bundles,
   },
+  // runDockedApp
```

Comment:
Could you add a blank line between the beginning of this function comment header and the end of the previous function? Thanks!

Reply:
Taken care of.